### PR TITLE
#14 make Makefiles cross-platform

### PR DESCRIPTION
**Pull request by CAMOBAP**
- #14

### CODE REVIEW

**Comment by CAMOBAP** [comment]
@ronaldtse @Intelligent2013 ready for review

Windows failed because we waiting for `isodoc` release of https://github.com/metanorma/isodoc/commit/ab536a1e53e82b8be0512e122347e2b94265bec0

**Comment by Intelligent2013** [comment]
@CAMOBAP 
Here in second line:
```
mn2stsDTD_NISO: $(DESTSTSXML) target/$(JAR_FILE) documents
	$(foreach f,$(DESTSTSXML),$(call mn2sts-check-type,$(f),dtd-iso))
```
looks like should be `dtd-niso`, not `dtd-iso`

**Comment by CAMOBAP** [comment]
@Intelligent2013 good catch, fixed

**Review by Intelligent2013 — APPROVED** [review]

File: .github/workflows/macos.yml [hunk 1]
```diff
@@ -24,7 +24,11 @@ jobs:
       - name: Setup prerequisites
         run: |
           # Install Maven
-          brew install maven
+          brew install maven yq
```
Comment (ronaldtse) [inline .github/workflows/macos.yml:31]: `yq` is duplicated?

File: Makefile [hunk 2]
```diff
@@ -47,18 +53,14 @@ deploy:
 documents/%.sts.html: documents/%.sts.xml saxon.jar
 	java -jar saxon.jar -s:$< -xsl:$(STS2HTMLXSL) -o:$@
 
-documents/%.sts.xml: documents/%.mn.xml target/$(JAR_FILE) | documents
+documents/%.sts.xml: documents/%.mn.xml | target/$(JAR_FILE) documents
```
Comment (ronaldtse) [inline Makefile:62]: The portion before `|` means if the file needs to exist. The portion after `|` means run the target regardless if it exists. Why do we need to change here?

**Pull request merged**